library/Vanilla/Embeds/TwitchEmbed.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPL-2.0
+ */
+
+namespace Vanilla\Embeds;
+
+use Exception;
+
+/**
+ * Twitch Embed.
```

Comment:
I'd like to see better class descriptions than this.

Something like `Class for parsing twitch URLs, fetching twitch OEmbed data, and rendering server side HTML for a twitch video embed`, is usefull. A comment that is the name of the class adds nothing but 3 extra lines.